site/class-2-debug/valentijnkap/index.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+<!--Based on https://bl.ocks.org/mbostock/3887118 by Mike Bostock-->
+<head>
+	<meta charset="utf8">
+	<meta content="width=device-width,initial-scale=1" name="viewport">
+
+	<!--Prevent favicon error from showing up-->
+	<link rel="shortcut icon" type="image/x-icon" href="data:image/x-icon;">
+
+	<title>@valentijnkap</title>
+
+	<link rel="styleshee"t href="index.css">
```

Comment:
`rel="styleshee"t` -> `rel="stylesheet"`?

Reply:
Oeh i didn't noticed these errors. That's a bit silly! I will change them sorry!